tests/sles4sap/sapconf.pm

```diff
@@ -106,12 +106,13 @@ sub verify_sapconf_service {
 
 sub run {
     my ($self) = @_;
-    my @tuned_profiles = is_sle('>=15') ?
-      qw(balanced desktop latency-performance network-latency network-throughput
-      powersave sapconf saptune throughput-performance virtual-guest virtual-host)
-      : qw(balanced desktop latency-performance network-latency
-      network-throughput powersave sap-ase sap-bobj sap-hana sap-netweaver
-      throughput-performance virtual-guest virtual-host);
+    my @tuned_profiles = qw(balanced desktop latency-performance network-latency network-throughput
+      powersave throughput-performance virtual-guest);
+    # Testing 'virtual-host' on a VM isn't very useful and can lead to sporadic timeout issues
+    push @tuned_profiles, 'virtual-host' unless (check_var('BACKEND', 'qemu'));
+    # Add some profiles depending of the OS version
+    my @suppl_profiles = is_sle('15+') ? qw(sapconf saptune) : qw(sap-ase sap-bobj sap-hana sap-netweaver);
+    push @tuned_profiles, @suppl_profiles;
```

Comment:
There is no need to define a new variable for this. You can do `push @tuned_profiles, is_sle('15+') ? qw(sapconfg saptune) : qw(sap-ase sap-bobj sap-hana sap-netweaver);` directly.

Reply:
Yes, and in fact I had this in my Lab previously... Something I didn't changed before pushing, thx!